src/execution/periodic_epoch.rs

```diff
@@ -108,6 +162,14 @@ where
                 assert!(output_cap.time() == state_update_cap.time());
                 let epoch = output_cap.time();
 
+                // Close `step_span` at each step, if previously open
```

Comment:
Since the change to a periodic span, I removed the `step` subdivision. We could actually introduce a proper one now, since the spans are handled in `run_until_done`, where we advance step by step, but with the "real" steps we get a lot of empty spans, since the flow spins through steps as fast as possible while waiting for things to do.

So for now we only have the periodic span, and subspans generated by the rest of the instrumentation.

Reply:
Ah right, I forgot about `Worker::step` being called "step". I am curious why we would have a lot of "empty steps" then. I thought in general workers would queue up their own work. But I guess that's a separate issue. Makes sense.

Reply:
That was in the wikistream example, where most of the time is spent waiting for new input, especially with more than one process, since we only generate the input on the first one. In those cases the worker keeps calling `step` and doing nothing since input is not received continuosly.